client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -484,7 +486,7 @@ class TransferDomainStep extends React.Component {
 			this.setState( prevState => {
 				const { submittingAvailability, submittingWhois } = prevState;
 
-				return { precheck: prevState.domain && ! submittingAvailability && ! submittingWhois };
+				return { precheck: prevState.domain && ! submittingAvailability && ! submittingWhois, domain };
```

Comment:
Minor readability nitpick: would you mind reformatting this object literal so that `precheck` and `domain` are on their own lines?  The domain property gets kind of list after that chunk of conditional logic.

Reply:
Haha, very true. I'll take care of that 👍.